packages/nimble-components/src/rich-text-editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -75,7 +76,7 @@ export class RichTextEditorPageObject {
      * To click a formatting button in the footer section, pass its position value as an index (starting from '0')
      * @param buttonIndex can be imported from an enum for each button using the `ButtonIndex`.
      */
-    public async clickFooterButton(buttonIndex: number): Promise<void> {
+    public async clickFooterButton(buttonIndex: ToolbarButton): Promise<void> {
```

Comment:
nit: none of these parameters should have the name `index` in them. that's an internal implementation detail we are hidding with the enum. just call them `button: ToolbarButton`, etc.

Reply:
Updated as per your suggestion.